traveler-ios-ui/TravelerKitUI/UIUtilities/FormView/FormView.swift

```diff
@@ -194,6 +238,64 @@ extension FormView: UICollectionViewDelegateFormLayout {
             fatalError("Unknown suppplementary element kind: \(kind)")
         }
     }
+
+    public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let _ = collectionView.cellForItem(at: indexPath) as? FormValueDisplayInputCell else {
+            return
+        }
+
+        UIView.setAnimationsEnabled(false)
```

Comment:
you shouldnt disable animations!...the goal was to fix the animations not disable them

Reply:
As I recall the issue was only around transparency...what needs to happen is the background on the cells need to be a solid color...thats it I think